demo/build.gradle.kts

```diff
@@ -0,0 +1,64 @@
+plugins {
+    java
+    id("com.diffplug.spotless")
+}
```

Comment:
Is it necessary to have a new build file (with mostly redundant entries)? Is there no way to put this under `src/demo/java` and "just" add that folder to be built/executed by the existing build?

In case it's not strictly necessary and only easier, it would be nice to know how much harder the other approach is because to me it seems more obvious (I really like the `demo` folder under `src`) and probably easier to maintain.